lineapy/api/models/pipeline.py

```diff
@@ -51,64 +48,9 @@ def __init__(
     def export(
         self,
         framework: str = "SCRIPT",
-        output_dir: Optional[str] = None,
+        output_dir: str = ".",
```

Comment:
I have one concern here which is it used to be that this output_dir would also be the exact directory where our files will be written, but now we are writing to `output_dir/pipeline_name`. 

This is fine for our tests since the return value of export and to_pipeline are still accurate, but for a user who might be throwing away the return value assuming the files are still in the same place this is different. 

cc @lionsardesai , @mingjerli

Reply:
I'm neutral about where we write the file, and I don't see a consensus default location that will satisfy everyone. We just need to clearly document where these files should be.

Reply:
As per team discussion, I am reverting to outputting pipeline files to output_dir simply (i.e., do not create a pipeline-specific subfolder). The main rationale is to be faithful to the user request, i.e., put files into the folder they specified.